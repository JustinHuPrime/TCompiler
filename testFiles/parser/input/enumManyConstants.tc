module foo;

enum bar {
  A,
  B,
  C,
};