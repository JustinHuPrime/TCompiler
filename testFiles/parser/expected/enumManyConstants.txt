testFiles/parser/enumManyConstants.tc (code):
FILE(1, 1, STAB(ENTRY(bar, ENUM(testFiles/parser/enumManyConstants.tc, 3, 1, CONSTANT(A, 0), CONSTANT(B, 1), CONSTANT(C, 2)))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), ENUMDECL(3, 1, ID(3, 6, bar, REFERENCES(testFiles/parser/enumManyConstants.tc, 3, 1)), ID(4, 3, A, REFERENCES(testFiles/parser/enumManyConstants.tc, 4, 3)), ID(5, 3, B, REFERENCES(testFiles/parser/enumManyConstants.tc, 5, 3)), ID(6, 3, C, REFERENCES(testFiles/parser/enumManyConstants.tc, 6, 3)), (null), (null), (null)))
